libminifi/test/unit/StringUtilsTests.cpp

```diff
@@ -91,5 +95,19 @@ TEST_CASE("TestStringUtils::testEnv5", "[test split classname]") {
 
   std::string expected = "hello world ";
 
-  REQUIRE(expected == StringUtils::replaceEnvironmentVariables(test_string));
+  REQUIRE(expected == replaceEnvironmentVariables(test_string));
+}
+
+TEST_CASE("TestStringUtils::testJoin", "[test string join]") {
+  std::set<std::string> strings = {"3", "2", "1"};
+  REQUIRE(join(",", strings) == "1,2,3");
```

Comment:
tests for empty empty sets, and empty delimiters would be nice here, too.

Reply:
Good idea, thanks, will add!